tests/tools/oesign/test-enclave/enclave/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(EDL_FILE ../helloworld.edl)
+
+# Use the edger8r to generate C bindings from the EDL file.
+add_custom_command(
+  OUTPUT helloworld_t.h helloworld_t.c helloworld_args.h
+  DEPENDS ${EDL_FILE} edger8r
+  COMMAND
+    edger8r --trusted ${EDL_FILE} --search-path ${PROJECT_SOURCE_DIR}/include
+    --search-path ${PLATFORM_EDL_DIR} --search-path ${CMAKE_CURRENT_SOURCE_DIR})
+
+add_executable(oesign_test_enc enc.c ${CMAKE_CURRENT_BINARY_DIR}/helloworld_t.c)
```

Comment:
I think this needs to be `add_enclave`

Reply:
@anakrish No, this is specifically is `add_executable` because `add_enclave` attempts to sign the enclave at build time, which is specifically not necessary/desirable for this set of tests.  This set of tests also explicitly do not benefit from LVI mitigations, or ability to span OP-TEE currently.

Reply:
I'd suggest adding a comment to note that difference.